utils/create_fixture.py

```diff
@@ -0,0 +1,271 @@
+                        help="MariaDB/MySQL user (by default = 'root')")
+    parser.add_argument("-p", "--password",
+                        default="",
+                        help="MariaDB/MySQL password (by default = '')")
+    parser.add_argument('-g', dest='debug_mode', action='store_true',
+                        default=False, help='Enable debug mode')
+    args = parser.parse_args()
+
+    return args
+
+
+def get_tables(db):
+    """Get a list of tables
+
+    :param db: MySQL connection
+    :return: table
+    """
+    tables = []
+    try:
+        logger.debug("MySQL: {}".format(QUERY_TABLES_SHOW))
+        cursor = db.cursor()
+        cursor.execute(QUERY_TABLES_SHOW)
+        data = cursor.fetchall()
+        for d in data:
+            tables.append(d[0])
+    except Exception as e:
+        logger.error("MySQL: {}: {}".format(QUERY_TABLES_SHOW, e))
+    finally:
+        logger.debug("Tables: {}".format(tables))
+        return tables
+
+
+def get_fixture(db, now, table):
+    """Create fixture given a table
+
+    :param db: MySQL connection
+    :param now: Datetime now
+    :param table: table
+    :return: fixture
+    """
+    fixture = []
+    model = get_model(table)
+    if not model:
+        return fixture
+
+    headers = get_headers(db, table)
+
+    try:
+        logger.debug("MySQL: {}".format(QUERY_SEARCH.format(table)))
+        cursor = db.cursor()
+        cursor.execute(QUERY_SEARCH.format(table))
+        data = cursor.fetchall()
+        count = 0
+        for d in data:
+            new = {
+                "fields": {},
+                "model": model
+            }
+            for index, h in enumerate(headers):
+                value = d[index]
+                if type(d[index]) == datetime:
+                    value = d[index].strftime(STRFTIME)
+                new['fields'][h] = value
+            new['fields']['created_at'] = now
+            new['fields']['last_modified'] = now
+            fixture.append(new)
+            count += 1
+        logger.debug("{} total items added {}".format(table, count))
+    except Exception as e:
+        logger.error(e)
+    finally:
+        return fixture
+
+
+def get_model(table):
+    """Get Django model like 'core.country'
+
+    :param table: table
+    :return: model
+    """
+
+    if table == "countries":
+        model = "core.country"
+    elif table == "domains_organizations":
+        model = "core.domain"
+    elif table == "enrollments":
+        model = "core.enrollment"
+    elif table == "identities":
+        model = "core.identity"
+    elif table == "matching_blacklist":
+        model = "core.matchingblacklist"
+    elif table == "organizations":
+        model = "core.organization"
+    elif table == "profiles":
+        model = "core.profile"
+    elif table == "uidentities":
+        model = "core.individual"
+    else:
+        model = None
+        logger.error("[ERROR] Table name unknown {}".format(table))
```

Comment:
Instead of doing this, create a dictionary `MODEL_MAPPING` like:

```
MODEL_MAPPING = {
    'countries' : 'core.country',
    'domains_organizations': 'core.domain'
    ...
}
```
Then you can do a `try..except` to catch the error if the table is unknown.